model-optimizer/extensions/front/CTCGreedyDecoderReplacement.py

```diff
@@ -89,3 +92,44 @@ def pattern(**kwargs):
 
     def replace_sub_graph(self, graph: Graph, match: dict):
         replace_ctc_greedy_decoder(graph, match)
+
+
+class CTCGreedyDecoderSingleReplacement(FrontReplacementSubgraph):
+    """
+    TensorFlow CTCGreedyDecoder produces output in a sparse tensor that is not supported by Inference Engine and
```

Comment:
Inference Engine and ---> Inference Engine, and
has different output --> has a different output
handle single TF CTCGreedyDecoder and warns user about another format of output ---> handles a single TF CTCGreedyDecoder and warns the user about another format of the output

Reply:
Done.